src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -20,13 +20,25 @@ trait ProxyRoutes extends UserInfoDirectives{ self: LazyLogging =>
   protected val tokenCookieName = "FCtoken"
 
   protected val proxyRoutes: Route =
+    pathPrefix("notebooks" / Segment / Segment / "api/localize") { (googleProject, clusterName) =>
```

Comment:
Is this a path prefix? If not it should be:
```
path("notebooks" / Segment / Segment / "api" / "localize") {
```

Reply:
Good catch, thanks. Copypasta :)